### PR TITLE
WRO-5337: RadioItem - icons in circle are bigger

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Some icons (eg. earphone, uninstall etc.), are bigger than the container for radio button 

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
All icons except circle (which is the default one), have `size="smallest"`

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-5337

### CODE REVIEW

File: RadioItem/RadioItem.module.less [hunk 1]
```diff
@@ -5,6 +5,18 @@
...
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		box-sizing: border-box;
+	}
+
+	.slotBefore {
+		display: flex;
+		align-items: center;
+	}
```
Comment (daniel-stoian-lgp) [inline RadioItem/RadioItem.module.less:18]: I couldn't find out why these 2 lines were added. Can you give more details?

Comment (adrian-cocoara-lgp) [reply RadioItem/RadioItem.module.less:18]: Well, by giving icon a width and height, when a `slotBefore` icon is used, the RadioItem icon and slotBefore icon are not aligned correctly. This was the fix I came up with

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**